armi/physics/fuelCycle/fuelHandlers.py

```diff
@@ -1243,6 +1263,10 @@ def _swapFluxParam(self, incoming, outgoing):
         incoming, outgoing : Assembly
```

Comment:
This flux and parameter swapping seems very strange to me still and is likely error prone. I really don't see why we _need_ to migrate the data from the assembly being discharged to the assembly being charged. Basically I foresee that this can introduce many bugs.

For example, consider a replacement of an assembly that is not of the same type (i.e., fuel with reflector or control). In this case, the power is flux and power are going to be immediately out dated and could be drastically incorrect. I think that removing this `_swapFluxParam` method is going to cause other things in the framework and code to break, but this entire process just seems unsafe.

Reply:
Removed ``_swapFluxParam`` entirely.

Reply:
@keckler  @ntouran  Can you guys weigh in on the solution used here, to completely remove `_swapFluxParam()`?

Thanks!

Reply:
I have never liked `_swapFluxParam()`. I think it could lead to some very confusing situations, in particular for everything that sits in the SFP. And I agree with all the issues that Jake stated.

That being said, I do think things will break if we just get rid of it completely.  But you probably would only see that in an integration test that (1) runs deterministic transport and (2) has shuffling. Might be other necessary features as well to see issues, but I'm not sure.

My _wish_ is to delete it, and barring that, to replace it with something smarter and more targeted.